Tests/CircumspectTests/Expectations/ExpectValueTests.swift

```diff
@@ -0,0 +1,40 @@
+//
+//  Copyright (c) SRG SSR. All rights reserved.
+//
+//  License information is available from the LICENSE file.
+//
+
+@testable import PillarboxCircumspect
+
+import Combine
+import XCTest
+
+private class Object: ObservableObject {
+    @Published var value = 0
+}
+
+final class ExpectValueTests: XCTestCase {
+    func testSingleValue() {
+        expectValue(from: Just([1, 2, 3].publisher))
+    }
+
+    func testMultipleValues() {
+        expectValue(from: Just([1, 2, 3].publisher))
+    }
```

Comment:
This test seems to check exactly the same thing as the test above (`testSingleValue`). We could try using a sequence here instead of `Just` to handle several emissions. 

Here is my suggestion:

```suggestion
    func testMultipleValues() {
        expectValue(from: Publishers.Sequence<[Int], Never>(sequence: [1, 2, 3]))
    }
```

Reply:
I don't know in which state I wrote this code 😅

There is a simpler alternative which requires less generics:

```swift
    func testMultipleValues() {
        expectValue(from: [1, 2, 3].publisher)
    }
```

This is what I pushed as a fix.